packages/react-native-executorch/common/rnexecutorch/models/ocr/Types.h

```diff
@@ -5,16 +5,16 @@
 #include <vector>
 
 namespace rnexecutorch {
+namespace ocr {
```

Comment:
Squash namespaces with `::` to namespace `rnexecutorch::ocr`, Same for ending comment